### ISSUE TITLE
Orient gui error

### ISSUE THREAD

**Issue by asl97** [Uberi/Minetest-WorldEdit#80]
```
2015-06-09 12:00:10: ERROR[main]: UNRECOVERABLE error occurred. Stopping server. Please fix the following error:
2015-06-09 12:00:10: ERROR[main]: .../bin/../mods/WorldEdit/worldedit_commands/init.lua:814: attempt to index local 'param' (a number value)
2015-06-09 12:00:10: ERROR[main]: stack traceback:
2015-06-09 12:00:10: ERROR[main]: 	.../bin/../mods/WorldEdit/worldedit_commands/init.lua:814: in function 'nodes_needed'
2015-06-09 12:00:10: ERROR[main]: 	.../bin/../mods/WorldEdit/worldedit_commands/safe.lua:21: in function 'func'
2015-06-09 12:00:10: ERROR[main]: 	.../bin/../mods/WorldEdit/worldedit_gui/functionality.lua:500: in function <.../bin/../mods/WorldEdit/worldedit_gui/functionality.lua:496>
2015-06-09 12:00:10: ERROR[main]: 	/home/(user)/(server)/bin/../builtin/game/register.lua:341: in function </home/(user)/(server)/bin/../builtin/game/register.lua:329>
```

**Comment by Uberi** [comment]
Hi @asl97, it would be helpful to know which WorldEdit version you're running. You can check this using the `/about` command.

**Comment by asl97** [comment]
@Uberi `/about` doesn't tell much other than 1.1,
imo, using git to get the commit would be better, that is 163dffccb38a1f4e37866d7232cecfb5238f7af7

**Comment by Uberi** [comment]
Should be fixed in fc037e9c82f34a6bf441dd738c1b10a510eb8b4d. Thanks for reporting!
